Sources/SwiftSyntax/generated/syntaxNodes/SyntaxNodesEF.swift

```diff
@@ -3806,23 +3798,17 @@ public struct FunctionEffectSpecifiersSyntax: SyntaxProtocol, SyntaxHashable, _L
     }
   }
   
-  /// The `throws` or `rethrows` keyword.
-  ///
-  /// ### Tokens
-  /// 
-  /// For syntax trees generated by the parser, this is guaranteed to be one of the following kinds:
-  ///  - `throws`
-  ///  - `rethrows`
-  public var throwsSpecifier: TokenSyntax? {
```

Comment:
This needs a compatibility layer.

Reply:
It's handled via the extension of the `EffectSpecifiersSyntax` protocol.